brave/brave-core
# Updated default ntp wallpaper image on Desktop

fix https://github.com/brave/brave-browser/issues/19127

This will be merged after resolving https://github.com/brave/brave-browser/issues/19129


Resolves 

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Launch browser with clean profile
2. Open NTP and check below `Dylan Malval`'s image is displayed till `NTP Background Images` component is loaded.

![Screenshot from 2021-11-01 12-12-08](https://user-images.githubusercontent.com/6786187/139616773-cb9bbea0-f5e2-4374-9789-cecd89d5c12d.png)
